packages/typespec-ts/package.json

```diff
@@ -67,12 +67,12 @@
     "eslint-plugin-require-extensions": "0.1.3"
   },
   "peerDependencies": {
-    "@azure-tools/typespec-azure-core": ">=0.34.0 <1.0.0",
-    "@azure-tools/typespec-client-generator-core": "0.35.0-dev.9",
-    "@typespec/compiler": ">=0.48.0 <1.0.0",
-    "@typespec/http": ">=0.48.0 <1.0.0",
-    "@typespec/rest": ">=0.48.0 <1.0.0",
-    "@typespec/versioning": ">=0.48.0 <1.0.0"
+    "@azure-tools/typespec-azure-core": ">=0.35.0 <1.0.0",
+    "@azure-tools/typespec-client-generator-core": ">=0.35.0 <1.0.0",
```

Comment:
As we are using peerDependency, do we really need to change this version number here?

Reply:
We have the compatibility issue caused by tcgc which would fail to compile. So we have to upgrade to the latest.